test/python/rest/submit_tests.py

```diff
@@ -0,0 +1,212 @@
+            logger.error(variable + ' environment variable not set')
+            result = False
+
+    return result
+
+@unittest.skipIf(not environment_variables_set() , "Streams environment variables not set")
+class RestSubmitTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+
+        # temporarily compile test application locally until REST API wrapper supports remote build service 
+        logger.warning('compile ' + applicationName)
+        result = os.system('sc -M ' + applicationName)
+        if not result==0: raise unittest.SkipTest(applicationName + ' failed to compile, exit code ' + str(result>>8) + ', signal ' + str(result&0xFF))
+
+        if not os.path.isdir('logs'):
+            os.mkdir('logs')
+
+        name = os.environ['STREAMING_ANALYTICS_SERVICE_NAME']
+
+        logger.warning('connect to IBM Cloud')
+        self.connection = streamsx.rest.StreamingAnalyticsConnection()
+
+        logger.warning('connect to service ' + name)
+        self.service = self.connection.get_streaming_analytics()
+
+        logger.warning('start service ' + name)
+        result = self.service.start_instance()
+        if not result['state']=='STARTED': raise unittest.SkipTest(name + ' service did not start')
+        if not result['status']=='running': raise unittest.SkipTest(name + ' service is not running')
+
+        logger.warning('check service ' + name)
+        instances = self.connection.get_instances()
+        if not len(instances)==1: raise unittest.SkipTest(name + ' service not found')
+        self.instance = instances[0]
+        if not self.instance.status=='running': raise unittest.SkipTest(name + ' service is not running')
+        if not self.instance.health=='healthy': raise unittest.SkipTest(name + ' service is not healthy')
+
+
+    def test_A_submit_with_defaults(self):
+
+        logger.warning('submit bundle ' + os.path.basename(applicationBundle))
+        result = self.service.submit_job(applicationBundle)
+        self.assertFalse('status_code' in result, 'submit_job() failed, status code and error description:' + str(result))
+        self.assertTrue('name' in result, 'submit_job() failed, no job name returned, result: ' + str(result))
+
+        logger.warning('let the job run a while')
+        time.sleep(runInterval)
+
+        logger.warning('check job ' + result['name'])
+        jobs = self.instance.get_jobs(result['name'])
+        self.assertTrue(len(jobs)>0, 'submit_job() failed, could not find job')
+
+        for job in jobs:
+            self.assertTrue(result['name']==job.name, 'submit_job() failed, could not find job')
+            self.assertTrue(job.status=='running', 'submit_job() failed, job is not running, status is ' + job.status + ', health is ' + job.health)
+            self.assertTrue(job.health=='healthy', 'submit_job() failed, job is not healthy, status is ' + job.status + ', health is ' + job.health)
+
+        for job in jobs:
```

Comment:
This seems to be testing `get_application_log`, just wondering why it was added, since there are existing tests for that.